plugins/CorePluginsAdmin/angularjs/form-field/form-field.directive.js

```diff
@@ -148,7 +148,7 @@
 
         function getTemplate(field) {
             var control = field.uiControl;
-            if (control === 'password') {
+            if (control === 'password' || control === 'url' || control === 'search' || control === 'email') {
```

Comment:
any reason for adding `search` here but not mentioning it in the changelog and adding a field constant?